ui/src/components/app-card/app-card.tsx

```diff
@@ -170,7 +170,11 @@ export const AppCard = ({
   };
 
   return (
-    <Box className="card" id={`card-${id}`} tabIndex={0}>
+    <Box
+      className={`card ${isAppCard ? '' : 'service'}`}
```

Comment:
What is this doing?

Reply:
Appliging a 'service' class to the service cards for unique styling ont he service ards. The service cards get a different height than the app cards.